libs/h2o/Makefile

```diff
@@ -0,0 +1,75 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=h2o
+PKG_VERSION:=2.2.6
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/h2o/h2o/tar.gz/v${PKG_VERSION}?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=f8cbc1b530d85ff098f6efc2c3fdbc5e29baffb30614caac59d5c710f7bda201
+
+PKG_MAINTAINER:=James Taylor <james@jtaylor.id.au
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+PKG_BUILD_DEPENDS:=ruby/host
+PKG_BUILD_PARALLEL:=1
+
+CMAKE_OPTIONS:= \
+	-DCMAKE_INSTALL_PREFIX=/usr \
+	-DCMAKE_INSTALL_LIBDIR=/usr/lib \
+	-DBUILD_SHARED_LIBS=ON \
+	-DWITH_MRUBY=OFF
+
+CMAKE_INSTALL:=1
+
+define Package/libh2o-evloop
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=H2O Library compiled with its own event loop
+  URL:=https://h2o.examp1e.net/
+  DEPENDS:=+libuv +libwslay +libopenssl +zlib +libyaml +ruby
+endef
+
+define Package/libh2o
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=H2O Library compiled with libuv
+  URL:=https://h2o.examp1e.net/
+  DEPENDS:=+libuv +libwslay +libopenssl +zlib +libh2o-evloop +libyaml
```

Comment:
This can be replaced with +libh2o-evloop

Reply:
I've removed the dependency on libuv from libh2o-evloop as well, so now libh2o only depends on libuv and evloop. I'm not sure if it actually depends on libh2o-evloop or not but that's how the debian packages do their dependencies.

Reply:
I've corrected the dependencies after reviewing the debian packages. The two implementations are completely separate, the only differentiating factor is that libh2o uses libuv for its event handler, and libh2o-evloop implements its own.